examples/ansible/roles/kubeone-reset/defaults/main.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019 The KubeOne Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kubeone_reset_destroyWorkers: true
+kubeone_reset_removeBinaries: false
+kubeone_versions_kubernetes: 1.14.1
+kubeone_clusterNetwork_podSubnet: 10.244.0.0/16
```

Comment:
Do we need all these variables for `reset`? The `reset` command mostly cares about hosts and fields needed to pass the validation.